proxylib/proxylib/types.go

```diff
@@ -86,6 +87,8 @@ func (r FilterResult) Error() string {
 		return "INVALID_ADDRESS"
 	case INVALID_INSTANCE:
 		return "INVALID_INSTANCE"
+	case UNKNOWN_ERROR:
+		return "UNKNOWN_ERROR"
 	}
-	return "UNKNOWN_ERROR"
+	return ""
```

Comment:
I would not change the default case to an empty string. Maybe cast to an int and format as a string?

Reply:
Good point, thanks.